test/h5test.c

```diff
@@ -114,11 +114,15 @@ const char *LIBVER_NAMES[] = {"earliest", /* H5F_LIBVER_EARLIEST = 0  */
 static H5E_auto2_t err_func = NULL;
 
 /* Global variables for testing */
-size_t   n_tests_run_g     = 0;
-size_t   n_tests_passed_g  = 0;
-size_t   n_tests_failed_g  = 0;
-size_t   n_tests_skipped_g = 0;
-uint64_t vol_cap_flags_g   = H5VL_CAP_FLAG_NONE;
+static int TestExpress_g     = -1; /* Whether to expedite testing. -1 means not set yet. */
```

Comment:
TestExpress functionality was moved back out of the testframe.c framework for more widespread usage in the future, but the global variable is once again hidden behind get/set functions so that tests don't have direct access to it.